packages/rum-core/src/domain/rumEventsCollection/view/trackViewMetrics.spec.ts

```diff
@@ -351,10 +353,17 @@ describe('rum track view metrics', () => {
       if (!('PerformanceObserver' in window) || !('supportedEntryTypes' in PerformanceObserver)) {
         pending('No PerformanceObserver support')
       }
+      originalSupportedEntryTypes = Object.getOwnPropertyDescriptor(PerformanceObserver, 'supportedEntryTypes')
       isLayoutShiftSupported = true
-      spyOnProperty(PerformanceObserver, 'supportedEntryTypes', 'get').and.callFake(() =>
-        isLayoutShiftSupported ? ['layout-shift'] : []
-      )
+      Object.defineProperty(PerformanceObserver, 'supportedEntryTypes', {
+        get: () => (isLayoutShiftSupported ? ['layout-shift'] : []),
+      })
+    })
+
+    afterEach(() => {
+      if (originalSupportedEntryTypes) {
+        Object.defineProperty(PerformanceObserver, 'supportedEntryTypes', originalSupportedEntryTypes)
```

Comment:
By bumping safari version, `supportedEntryTypes` is now available ... but not in the same way than the other browsers 😅 
So, since there is no `get` method on his property descriptor, I had to find an alternate strategy.

Reply:
I see, thanks!